### ISSUE TITLE
"Go To Definition" not working on Mac

### ISSUE THREAD

**Issue by Jongkeun** [microsoft/vscode#75550]
I installed vscode on my Mac.
At first I was embarrassed by the fact that Windows and shortcuts were very different.
I think these are the problems I have to adapt.
But 'Go to Definition' function should work properly.
I saw there were many issues about `golang`. 
But I think it's different in my case.
I tested it with nothing in the project too. But it doesn't work at all.
I tested the same case in Windows too, but it worked well.
Let me know what I should check and do to solve this problem.

- VSCode Version: 1.35.1
- OS Version: Mojave 10.14.5 and High Sierra the last version

Steps to Reproduce:

1. type this code in `test.js`
```
let test = 1;
let result = 1 + test;
```
2. set focus at `test` in second line.
3. `F12` or `cmd+click` or `Go -> Go to Definition` click

Does this issue occur when all extensions are disabled?: Yes

**Comment by Jongkeun** [comment]
Hey @Tyriar. Are you serious?
I uninstalled all of extension.

**Comment by mjbvz** [comment]
I can't reproduce this issue. Have you perhaps disabled the builtin `TypeScript and JavaScript Language Features` extension? In the extensions view, search `@builtin JavaScript` to confirm it is still enabled.

**Comment by Jongkeun** [comment]
I had a state 'disabed Extension'.
Because I thought this problem was caused by extension.
But it was still not working.
So 'Ah whatever I do, it's not working. I gonna use extensions.'
And I changed the state.
It works well after I changed to 'Enable Extension'.
And I changed to 'Disable Extension' again.
It works well too.
I think it's related in initial value of the state.

Anyways, thank you for your care, @mjbvz .
